src/test/crimson/seastore/test_transaction_manager.cc

```diff
@@ -774,7 +798,8 @@ TEST_F(tm_single_device_test_t, allocate_lba_conflict)
       t,
       ADDR,
       SIZE,
-      'a');
+      'a',
+      tm_config.default_placement_hint);
```

Comment:
I mean that this argument is passed into alloc_extent identically everywhere in this file.  Remove the hint argument from alloc_extent entirely and set the default there.